dataretrieval/wqp.py

```diff
@@ -468,3 +486,19 @@ def _alter_kwargs(kwargs):
     kwargs['mimeType'] = 'csv'
 
     return kwargs
+
+def _three_point_oh_warn():
```

Comment:
```suggestion
def _warn_v3_profiles_outage():
```
In general, recommend putting "warn" first so like functions are grouped alphabetically.